packages/adapter-vercel/index.js

```diff
@@ -219,6 +226,20 @@ const plugin = function (defaults = {}) {
 				group.routes.push(route);
 			}
 
+			if (ignored_isr.size) {
+				builder.log.warn(
+					`\nWarning: The following routes have an ISR config which is ignored because the route is prerendered:`
```

Comment:
I made this a warning, thought about whether or not this is an error, but I think that would be breaking - maybe in the next major? Or would this be more annoying than helpful if it was an error?

Reply:
I think a warning is probably fine, you could have a prerendered leaf within an ISR'd group, for example. we can see if people trip up on it in practice